### PR TITLE
add esp32-cam example

### PR DESCRIPTION
**Pull request by Kezii**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ You can usually find community members (including some employees of Espressif) i
 - [self-balancing robot](https://github.com/tana/atom-motion-balance) - Self-balancing robot using Atom Matrix and Atom Motion. [See details](https://zenn-dev.translate.goog/tana_ash/articles/5a458538cd9204?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=wapp).
 - [twatch-idf-rs](https://github.com/pyaillet/twatch-idf-rs) - Experimental firmware for the Lilygo T-Watch
 - [wrover-experimental](https://github.com/JurajSadel/wrover-experimental) - Display line directions and departure times of public transport vehicles in Brno, CZ.
+- [esp32cam-rs](https://github.com/Kezii/esp32cam_rs/tree/master) - Examples of Rust on ESP32-CAM, including a fully-featured Telegram bot capable of taking pictures.
```
Comment (SergioGasquez) [inline README.md:92]:
```suggestion
- [esp32cam-rs](https://github.com/Kezii/esp32cam_rs) - Examples of Rust on ESP32-CAM, including a fully-featured Telegram bot capable of taking pictures.
```

**Review by SergioGasquez — COMMENTED** [review]
Thanks for your contribution! Do you mind-expanding the repos README so its more beginner friendly? A few ideas:
- Add the run command for each example
- Add the txt files required for each example (ssdi, pass, bot token...)  
  
 Again, thanks for this amazing examples!

**Comment by Kezii** [comment]
Hello,
I did the requested changes to the repo, let me know what you think

Thank you

**Review by SergioGasquez — APPROVED** [review]
LGTM! Thanks for the contribution!

**Pull request merged**